src/components/Sidebar/Sidebar.test.jsx

```diff
@@ -43,10 +34,7 @@ describe("Sidebar", () => {
   describe("render", () => {
     describe("snapshots 📸", () => {
       it("Sidebar should match snap shot", () => {
-        matches(<Sidebar />);
-      });
-      it("Sidebar should match snap shot if hide props is true", () => {
```

Comment:
??

Reply:
hide doesn't exists

Reply:
My poor `hide` prop

`<Navbar className="billSnap-SideBar" style={ this.props.hide ? { display: 'none' } : {}}>`

Reply:
this is the first line in SideBar render